web/themes/custom/usagov/templates/includes/benefits-search.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * Included in the basic node page template and has all the same variables available.
+ */
+ #}
+{% set lang = node.langcode.value %}
+
+{% if lang == 'en' %}
+{%   set life_event_title = "Benefit finder tool" %}
+{%   set search_block = "benefits_facets_block" %}
+{% elseif lang == 'es' %}
+{%   set life_event_title = "Encuntre beneficios en estas etapas de su vida" %}
```

Comment:
We definitely want to get the titles out into a place where the content editors can get at them.